### ISSUE TITLE
openssl passwd in rhel 7 does not support sha512 based password algorithm

### ISSUE THREAD

**Issue by henrywang** [weldr/welder-web#631]
Adding user account does not work on rhel 7 because openssl passwd in rhel 7 does not support sha512 based password algorithm.

https://github.com/weldr/welder-web/blob/12584fb192dd4ce9f8f874e07d5ebf37e98c6fed/components/Modal/UserAccount.js#L185

`openssl passwd` output:
```shell
[root@m1 ~]# openssl passwd -6 -stdin
Usage: passwd [options] [passwords]
where options are
-crypt             standard Unix password algorithm (default)
-1                 MD5-based password algorithm
-apr1              MD5-based password algorithm, Apache variant
-salt string       use provided salt
-in file           read passwords from file
-stdin             read passwords from stdin
-noverify          never verify when reading password from terminal
-quiet             no warnings
-table             format output as table
-reverse           switch table columns
```

**Comment by martinpitt** [comment]
Unfortunately it's ridiculously complicated to encrypt a password on the command line; `openssl passwd` is pretty much the only way; `chpasswd` knows how to encrypt, but only operates on the /etc/shadow file, not on stdout. There are a [few ways using python or perl](https://serverfault.com/questions/330069/how-to-create-an-sha-512-hashed-password-for-shadow), but these all mean additional dependencies, or mucking around with python2-only vs. python3-only distros.

With that much trouble, I think it would actually make more sense to do it on the JavaScript side with [bcrypt](https://www.npmjs.com/package/bcrypt).

**Comment by martinpitt** [comment]
Fixed in PR #661, just got wrongly referenced in the commit message. (@henrywang : This should be "Fixes #xxxx")
